src/main/java/io/github/jazorp/Validators.java

```diff
@@ -33,6 +37,25 @@
     // =====================
     // Validators
 
+	public static ValidationThunk memberOf(String field, final Object value, Iterable<Object> values) {
+		Collection<Object> collection = null;
+		if (values instanceof Collection) {
+			collection = (Collection<Object>) values;
+		} else {
+			collection = new ArrayList<>();
+			values.forEach(collection::add);
+		}
+		return memberOf(field, value, collection);
+	}
+
+	public static ValidationThunk memberOf(String field, final Object value, Object... values) {
+		return memberOf(field, value, Arrays.asList(values));
+	}
+
+	public static ValidationThunk memberOf(String field, final Object value, Collection<Object> values) {
```

Comment:
`Collection` extends `Iterable` so this implementation is effectively redundant.